hub/tx_init/rows/acc_recipient_value.tex

```diff
@@ -3,11 +3,11 @@
 	\left\{ \begin{array}{lcl}
 		\accAddress \high _{i + \locTxInitRoffAccDoingRecipientValueReception} & = & \locTxInitRecipientAddressHi \\
 		\accAddress \low  _{i + \locTxInitRoffAccDoingRecipientValueReception} & = & \locTxInitRecipientAddressLo \\
-		\multicolumn{3}{l}{
+		\multicolumn{4}{l}{
 			\accTrimAddress
 			{i}{\locTxInitRoffAccDoingRecipientValueReception}
 			{\locTxInitRecipientAddressHi}
-			{\locTxInitRecipientAddressLo}} \vspace{2mm} \\
+			{\locTxInitRecipientAddressLo}} \vspace{2mm}  \quad (\trash) \\
```

Comment:
This, too, isn't trash. We require it for the same reason as in hub/tx_init/rows/acc_sender_gas.tex + the fact that we actively disallow the recipient to be a precompile.